frontend/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+import { API_BASE_URL } from "@/config";
+
+import { Footer as IFooter, Logo as ILogo } from "@types/Footer";
+
+export const Footer: React.FC<IFooter> = ({disclaimer, logos, privacy}) => {
+    return (
+        <div className="aha__footer">
+            <div className="disclaimer" dangerouslySetInnerHTML={{
+                __html: disclaimer,
+                }}
+            />
+            <div className="logos">
+                {logos.map((logo: ILogo, index: number) => (
+                    <a
+                        href={logo.href}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                        key={index}
+                    >
+                        <img src={API_BASE_URL + logo.file} alt={logo.alt} />
+                    </a>
+                ))};
```

Comment:
This semicolon results into a weird layout:

```suggestion
                ))}
```

<img width="1042" alt="image" src="https://github.com/Amsterdam-Music-Lab/MUSCLE/assets/8208970/2a6e8b83-5e8e-415a-abb8-fa5484895634">